skypy/pipeline/tests/test_items.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+
+def test_item():
+    from skypy.pipeline import Pipeline
+    from skypy.pipeline._items import Item
+
+    # set up a mock pipeline
+    pipeline = Pipeline({})
+
+    # construct the base class
+    item = Item()
+
+    # check `infer` method
+    item.infer({})
+
+    # check `depend` method
+    deps = item.depend(pipeline)
+    assert isinstance(deps, list)
+
+    # check `evaluate` method
+    val = item.evaluate(pipeline)
+    assert val is None
+
+
+def test_call():
```

Comment:
Could we modify this test slightly to provide one of the arguments as a pipeline variable (instead of in context) and then check that Call.depends returns the appropriate Ref?